nemo/collections/asr/metrics/rnnt_wer.py

```diff
@@ -177,13 +178,18 @@ def rnnt_decoder_predictions_tensor(
                 decoded_hyps = self.decode_hypothesis(n_hyps)  # type: List[str]
                 hypotheses.append(decoded_hyps[0])  # best hypothesis
                 all_hypotheses.append(decoded_hyps)
-
-            return hypotheses, all_hypotheses
+            if return_hypotheses:
+                return hypotheses, all_hypotheses
+            return [h.text for h in hypotheses], [h.text for hh in all_hypotheses for h in hh]
```

Comment:
Hmm I think it would be cleaner to do this instead - 
```python
best_hyp_text = [h.text for h in hypotheses]
all_hyp_text = [h.text for hh in all_hypotheses for h in hh]
return best_hyp_text, all_hyp_text 
```